src/Storefront/Resources/views/storefront/layout/navbar/navbar.html.twig

```diff
@@ -1,10 +1,15 @@
 {% block layout_navbar %}
     <div class="container">
+        {% set navbarOptions = {
+            activeRootId: shopware.navigation.pathIdList[0],
```

Comment:
maybe we should add a more descriptive getter method in `\Shopware\Storefront\Framework\Twig\NavigationInfo`. Instead of using the first element from the array :thinking:

Reply:
@mitelg I added a method

Reply:
I removed it again because we not use the pathIdList like the inline styling.